src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -277,57 +338,66 @@ TEST_F(PolicyManagerImplTest, LoadPT_SetPT_PTIsLoaded) {
   EXPECT_CALL(*cache_manager, SaveUpdateRequired(false));
   EXPECT_CALL(*cache_manager, TimeoutResponse());
   EXPECT_CALL(*cache_manager, SecondsBetweenRetries(_));
+  EXPECT_CALL(*cache_manager, GetHMIAppTypeAfterUpdate(_));
 
-  TEST_F(PolicyManagerImplTest2, NextRetryTimeout_ExpectTimeoutsFromPT) {
-    // Arrange
-    std::ifstream ifile("sdl_preloaded_pt.json");
-    Json::Reader reader;
-    Json::Value root(Json::objectValue);
-    if (ifile.is_open() && reader.parse(ifile, root, true)) {
-      Json::Value seconds_between_retries = Json::Value(Json::arrayValue);
-      seconds_between_retries =
-          root["policy_table"]["module_config"]["seconds_between_retries"];
-      uint32_t size = seconds_between_retries.size();
-      CreateLocalPT("sdl_preloaded_pt.json");
-
-      uint32_t waiting_timeout = 0u;
-
-      for (uint32_t retry_number = 0u; retry_number < size; ++retry_number) {
-        waiting_timeout += seconds_between_retries[retry_number].asInt();
-        waiting_timeout += manager->TimeoutExchange();
-
-        // it's in miliseconds
-        EXPECT_EQ(waiting_timeout * date_time::DateTime::MILLISECONDS_IN_SECOND,
-                  manager->NextRetryTimeout());
-      }
-    }
-  }
+  EXPECT_TRUE(manager->LoadPT("valid_sdl_pt_update.json", msg));
+}
 
-  TEST_F(PolicyManagerImplTest,
-         RequestPTUpdate_SetPT_GeneratedSnapshotAndPTUpdate) {
+TEST_F(PolicyManagerImplTest2, DISABLED_NextRetryTimeout_ExpectTimeoutsFromPT) {
+  // Arrange
+  std::ifstream ifile("sdl_preloaded_pt.json");
+  Json::Reader reader;
+  Json::Value root(Json::objectValue);
 
-    // arrange
-    ::utils::SharedPtr<::policy_table::Table> p_table =
-        new ::policy_table::Table();
+  if (ifile.is_open() && reader.parse(ifile, root, true)) {
+    Json::Value seconds_between_retries = Json::Value(Json::arrayValue);
+    seconds_between_retries =
+        root["policy_table"]["module_config"]["seconds_between_retries"];
+    uint32_t size = seconds_between_retries.size();
 
-    // assert
-    EXPECT_CALL(*cache_manager, GenerateSnapshot()).WillOnce(Return(p_table));
+    CreateLocalPT("sdl_preloaded_pt.json");
```

Comment:
@orhan-mehmedov as I remember this method is empty, are you sure that it is to call it?